honeybee_3dm/face.py

```diff
@@ -70,12 +70,14 @@ def import_objects_with_config(rhino3dm_file, layer, tolerance, *,
                     ' visible on rhino canvas, switch to shaded mode, and save the file.'
                     )
             except AssertionError:
-                raise AssertionError(
+                warnings.warn(
                     f'Could not create a face for object of ID {obj.Attributes.Id}'
                     ' Please reduce the unit tolerance value in rhino, save the file and'
                     ' try again. You might need to repeat this more than once if the'
                     ' face is too small for the unit tolerance selected.'
                     )
+                continue
```

Comment:
You can just define this long message once with blank `{}` and use `.format({obj.Attributes.Id})` to fill in the object ID in the different places.

Reply:
@chriswmackey, I am not sure if I understand your suggestion. Can you share an example?

Reply:
@chriswmackey is suggesting reusing the text as a template.

```python
warn_msg = 'Could not create a face for object of ID {object_id}' \
                    ' Please reduce the unit tolerance value in rhino, save the file and' \
                    ' try again. You might need to repeat this more than once if the' \
                    ' face is too small for the unit tolerance selected.'
```

Then when you want create a warning you can do.

```python
warning.warn(warn_msg .format(object_id=obj.Attributes.Id)
```

This will avoid writing the same long message twice. I used a variable name (object_id) to make it more readable but you can also use a blank `{}` as suggested by Chris. In that case it will look like this.


```python
warning.warn(warn_msg .format(obj.Attributes.Id)
```